Default.sublime-keymap

```diff
@@ -0,0 +1,3 @@
+[
+    { "keys": ["super+shift+o"], "command": "side_bar_folders_quick_switch" }
```

Comment:
As a compromise, you could expose the command in the command palette to give users something useable out of the box and provide an example to get immediate access via a keybinding.